### PR TITLE
Update Add-music.md

### PR DESCRIPTION
**Pull request by saarthakchats**
We had collaborated on this work, pls merge.
Contains improvements in task description

I hope this works for "Enabling a new task" and also "Collaborate on PR"

### CODE REVIEW

**Comment by niccokunzmann** [comment]
So, when the issue is closed, you can claim your part in the enable a task task. Collaborating on a PR will only work if both your commits are in one PR and you both agree to collaborate.

**Comment by niccokunzmann** [comment]
The idea of enabling a task is to provide a uniform way of adding something to the game. e.g. #161 or #173. Then, we have an easy task which has most of the focus on the creative work and less on changing existing things. So it is with the background. The issue needs to be closed, I agree to post the task and then, people can use 99% of their time to record music and 1% to upload it to the game. Adding parts of the issue to the task will not create this ease of use. What do you think?

File: tasks/Add-music.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Requirements:
 - It should be music relevant to the theme, try not to add any songs
 - Please make sure to use the existing API.
 - The music should be your own work. Please make sure you have the rights to any music that you add.
+- the music of any tile stops playing when the player leaves not moves
```
Comment (niccokunzmann) [inline tasks/Add-music.md]: This should be done by the API but it is good to add, we should refer to the API. The documentation is merged.

**Comment by saarthakchats** [comment]
You are right

File: tasks/Add-music.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Requirements:
...
+  choose the background music randomly
+- add a loudness to the background music in the object you store it. Some music may be loud, some music is quite silent. If we add         music   from different people, we want to control that in the game as they make music of different volume.
+  document different open-source software in the task that we can use.
+  recording: audacity
+- When all is done, we can create a new feature task to show a configuration menu for the two music types (on off/loudness)
```
Comment (niccokunzmann) [inline tasks/Add-music.md]: this is not part of the task.

File: tasks/Add-music.md [hunk 3]
```diff
@@ -13,6 +13,12 @@ Requirements:
 - It should be music relevant to the theme, try not to add any songs
 - Please make sure to use the existing API.
 - The music should be your own work. Please make sure you have the rights to any music that you add.
+- the music of any tile stops playing when the player leaves not moves
+  choose the background music randomly
+- add a loudness to the background music in the object you store it. Some music may be loud, some music is quite silent. If we add         music   from different people, we want to control that in the game as they make music of different volume.
+  document different open-source software in the task that we can use.
```
Comment (niccokunzmann) [inline tasks/Add-music.md]: this can be formulated differently. as editing the task is not part of the task.

**Comment by niccokunzmann** [comment]
@saarthakchats I commented. What do you think?

**Comment by saarthakchats** [comment]
Yes it won't create any ease of use

**Comment by niccokunzmann** [comment]
I will merge now. If you like to undo the changes, you can click the revert button.

**Pull request merged**